zebra-chain/src/orchard/tree.rs

```diff
@@ -170,6 +173,26 @@ impl ZcashDeserialize for Root {
 #[derive(Copy, Clone, Debug, Eq, PartialEq)]
 pub struct Node(pallas::Base);
 
+impl Node {
+    /// Calls `to_repr()` on inner value.
+    pub fn to_repr(&self) -> [u8; 32] {
+        self.0.to_repr()
+    }
+
+    /// Converts a byte representation of a field element into an element of
+    /// this prime field without checking if the input is not canonical
+    pub fn from_repr_unchecked(bytes: &[u8]) -> Self {
```

Comment:
Is this needed?

Reply:
Nope, removed in https://github.com/ZcashFoundation/zebra/pull/7371/commits/10289312a42a4d2793845c121122477a1cbbaddd.